packages/ag-charts-community/project.json

```diff
@@ -127,6 +127,18 @@
         "output": "dist/packages/ag-charts-community/resolved-interfaces.AUTO.json"
       }
     },
+    "docs-resolved-interfaces-patched": {
```

Comment:
Why do we need a new/distinct build target?

Reply:
It was mainly for comparing, during debugging, but can remove